sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/DotCoverReportParserTest.java

```diff
@@ -101,6 +147,18 @@ public void valid() throws Exception {
       .startsWith("dotCover parser: found coverage for line '12', hits '0' when analyzing the path '");
   }
 
+  @Test
+  public void valid_big() throws Exception {
+    Coverage coverage = new Coverage();
+    new DotCoverReportParser(alwaysTrue).accept(new File("src/test/resources/dotcover/valid_big.html"), coverage);
```

Comment:
do you think it's a good idea to, inside the `valid_big.html` file, replace the contents inside `Add10000()` with a placeholder, and generate the `value = value + 1;` statements here in the test? and create a temporary file from the template + code?

because having a 10K LOC file is difficult to handle: 
-  both to load (the IDE + plugins may be very slow), also github
- and to review historical diffs

Reply:
The file review is one-off task. I don't expect any changes there in a future. So I don't think we need complicated scaffolding in this case. The plain file is simple and 100% sure to be big.